src/components/Dialog/Dialog.less

```diff
@@ -53,6 +53,13 @@
 		max-height: 60vh; // fix for IE 11 bug, otherwise the body doesn't shrink properly
 	}
 
+	&-body-no-gutters {
+		flex: 1 1 auto;
```

Comment:
If you keep the `&-body` class and conditionally add the `&-body-no-gutters` class you should be able to make this block just have `padding: 0` I think.

Reply:
or you'll probably want `&-body.&-body-no-gutters {`